packages/react-components/src/utils/setupConfiguration.ts

```diff
@@ -116,6 +139,8 @@ async function saveConfiguration(
         PasswordKeys.INGEST_METADATA_ONLY,
         `${isIngestingMetadataOnly}`
     );
+    await savePassword(storagePasswords, PasswordKeys.SEVERITIES_FILTER, `${severitiesFilter}`);
```

Comment:
I think we should probably only be storing the API key in storage_passwords. Everything else really should be in KVStore. Thoughts?

Reply:
Yep I will change that

Reply:
Thanks to Splunk, we can't do it using their SDK and we use their authentication wrapper to do requests